setup/modelfactory.class.inc.php

```diff
@@ -1834,6 +1834,13 @@ public function GetNodes($sXPath, $oContextNode = null, $bSafe = true)
 	{
 		return $this->oDOMDocument->GetNodes($sXPath, $oContextNode, $bSafe);
 	}
+
+	/**
+	 * @return mixed
+	 */
+	public function getRootDirs() {
```

Comment:
```suggestion
	 * @return mixed
	 * @since 3.0.0
	 */
	public function GetRootDirs() {
```

Can you just add a comment on what you mean by root dirs? Root dirs of the iTop app. ?